packages/DialogActions/src/DialogActions.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "../../Button/src/Button";
+
+const propTypes = {
+  /** Optional custom classes. */
+  className: PropTypes.string,
+
+  /** If a cancel button should be shown. */
+  hasCancel: PropTypes.bool,
+
+  /** If a primary action button should be shown. */
+  hasConfirm: PropTypes.bool,
+
+  /** If a secondary generic button should be shown. */
+  hasDecline: PropTypes.bool,
+
+  /** If the primary button is a destructive action. */
+  isDestructive: PropTypes.bool,
+
+  /** If the primary and secondary buttons are disabled. */
+  isDisabled: PropTypes.bool,
+
+  /** Text for cancel button. */
+  labelCancel: PropTypes.string,
+
+  /** Text for primary action button. */
+  labelConfirm: PropTypes.string,
+
+  /** Text for a secondary generic button. */
+  labelDecline: PropTypes.string,
+
+  /** Function to call when cancel button is clicked. */
+  onCancel: PropTypes.func,
+
+  /** Function to call when primary button is clicked. */
+  onConfirm: PropTypes.func,
+
+  /** Function to call when secondary button is clicked. */
+  onDecline: PropTypes.func,
+};
+
+const defaultProps = {
+  className: "",
+  hasCancel: true,
+  hasConfirm: true,
+  hasDecline: true,
+  isDestructive: false,
+  isDisabled: false,
+  labelCancel: "Cancel", // TODO: i18n
+  labelConfirm: "Save", // TODO: i18n
+  labelDecline: "Don't Save", // TODO: i18n
```

Comment:
Let's update this to use the `I18n`. You can find out how to do this by looking up areas in the code where it uses `I18n.t`

Reply:
done